### ISSUE TITLE
Remeber settings between sessions

### ISSUE THREAD

**Issue by tooomm** [Senryoku/MTGADraft#323]
Game settings are not saved and are lost when reloading the site.
There are certain settings that the host always has to adjust according to his playgroup or personal preferences, e.g. log distribution.

App settings like disabled sound or enabled pick on double click are saved already.

**Comment by Senryoku** [comment]
- eb94f5207390a0e2d09704dd97df593e6abc41c2 Saves a subset of the session settings to be reapplied automatically when the user create a new session. See the list of saved settings bellow. I limited it to settings that are relevant to the personal preferences for the user, not to a particular game mode. Is there other settings I should include? 
 - b9a4eb26f240981f5f404abda64a11f9b057356a Adds a reset session settings button (I felt like this was necessary with the previous change)
![image](https://user-images.githubusercontent.com/1338143/171847114-01df3f73-9515-41ee-83c3-3cad88b6f4c5.png)


Theses patches can be tested at http://mtgadraftbeta.herokuapp.com/

Currently saved session settings:
```
randomizeSeatingOrder
disableBotSuggestions
maxTimer
ignoreCollections
foil
personalLogs
draftLogRecipients
```

**Comment by tooomm** [comment]
I think you covered the important pieces. 👍 
The reset is also nice.

Players do not enter a fresh session now though, when going to the side. It's remembered as well.

**Comment by Senryoku** [comment]
> Players do not enter a fresh session now when going to the side, though. It's remembered as well.
> But that seems to be only an issue on beta.

Sorry, I don't understand, what's the problem?

**Comment by tooomm** [comment]
Nevermind. It was due to my test browser not having custom values set and it received new random user names and new random session id's on every reconnect. As soon as you change the session id (append a random character for example), you'll always rejoin/reopen that same session when entering the page - even via the default URL with no session id attached.
I did not understand right away why it's not giving me new values suddenly.
